assets/scss/modules/_side-navigation.scss

```diff
@@ -194,6 +210,7 @@ Styleguide Side Navigation.overlapping
 .side-nav__rhs-content {
```

Comment:
> whats rhs ?

as discussed being verbose aids in understanding `right-hand-side` feels easier to understand than `rhs`